src/main/plugin/dcat-ap/layout/config-editor.xml

```diff
@@ -343,6 +343,28 @@
         </snippet>
       </template>
     </for>
+
+    <!-- DCAT-AP v2 compatibility -->
+    <for name="dct:isVersionOf" templateModeOnly="true" forceLabel="true" label="key">
+      <template>
+        <values>
+          <key label="key" xpath="@rdf:resource" tooltip="dct:isVersionOf" required="true"/>
+        </values>
+        <snippet>
+          <dct:isVersionOf rdf:resource="{{key}}"/>
+        </snippet>
+      </template>
+    </for>
+    <for name="dct:hasVersion" templateModeOnly="true" forceLabel="true" label="key">
+      <template>
+        <values>
+          <key label="key" xpath="@rdf:resource" tooltip="dct:hasVersion" required="true"/>
+        </values>
+        <snippet>
+          <dct:hasVersion rdf:resource="{{key}}"/>
+        </snippet>
+      </template>
+    </for>
```

Comment:
Is there a need for keeping DCAT-ap 2 compatibility in the editor ?  
Since DCAT 2 and 3 are pretty similar, maybe the `update-fixed-info.xsl` should automatically migrate any DCAT-ap 2 records being edited.